tests/testthat/test_MzTab.R

```diff
@@ -11,7 +11,7 @@ test_that("MzTab creation and accessors", {
     expect_null(show(xx))
     ## Accessors
     expect_is(metadata(xx), "list")
-    expect_identical(length(metadata(xx)), 64L)
+    expect_identical(length(metadata(xx)), 65L)
```

Comment:
Turns out that the previous implementation without `headers=FALSE` swallowed the first line of the MTD section. After fixing that there is one more metadata line.